### PR TITLE
Docsp 10662 v3.6

### PR DESCRIPTION
**Pull request by biniona-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-10662

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/e1cd138/node/docsworker-xlarge/DOCSP-10662-V3.6/fundamentals/connection/#connection-options

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/fundamentals/connection.txt [hunk 1]
```diff
@@ -143,11 +143,32 @@ URI to specify the behavior of the client.
      - Specifies the logger level used by the driver. Valid choices are:
        ``error``, ``warn``, ``info``, and ``debug``.
 
+   * - **maxPoolSize**
+     - integer
+     - ``10``
+     - **This option requires that you set the useUnifiedTopology option to true**.
```
Comment (terakilobyte) [inline source/fundamentals/connection.txt:149]:
```suggestion
     - You must set the ``useUnifiedTopology`` option to **true** to use this option.
```

File: source/fundamentals/connection.txt [hunk 2]
```diff
@@ -214,7 +235,15 @@ URI to specify the behavior of the client.
...
-       override this behavor by setting the value to ``false``.
+       override this behavior by setting the value to ``false``.
+
+   * - **useUnifiedTopology** 
+     - boolean 
+     - ``true``
+     - Specifies whether to use the **unified topology** layer. To
+       learn what the unified topology layer is and why it was introduced,
+       see `this document <https://github.com/mongodb/node-mongodb-native/blob/3.6/docs/reference/content/reference/unified-topology/index.md>`__
+       produced by the MongoDB Node driver engineering team.
```
Comment (terakilobyte) [inline source/fundamentals/connection.txt:246]:
```suggestion
       . 
```

File: source/fundamentals/connection.txt (hunk 1 above)
Comment (terakilobyte) [reply source/fundamentals/connection.txt:149]: This also applies to the other instances.

**Review by terakilobyte — CHANGES_REQUESTED** [review]
Looks good mod a few suggestions.

File: source/fundamentals/connection.txt (hunk 2 above)
Comment (biniona-mongodb) [reply source/fundamentals/connection.txt]: Thanks for the note. Discussed this suggestion on slack, and I agree it doesn't add much for users to mention the engineering team. Fix: https://github.com/mongodb/docs-node/pull/149/commits/6601707dfc483757b82c83aa2ca4ce3e001450f6

File: source/fundamentals/connection.txt (hunk 1 above)
Comment (biniona-mongodb) [reply source/fundamentals/connection.txt]: Thanks for the note. I agree that this phrasing is more linear, more imperative, and easier to understand. Will try and keep this in mind going forward.

Fix: https://github.com/mongodb/docs-node/pull/149/commits/e1cd138fc81b04e757e426e99a2027e8d6a69905

**Pull request merged**